components/brave_wallet/browser/brave_wallet_constants.h

```diff
@@ -409,14 +368,10 @@ constexpr webui::LocalizedString kLocalizedStrings[] = {
      IDS_BRAVE_WALLET_CONFIRM_TRANSACTION_TOTAL},
     {"braveWalletConfirmTransactionGasFee",
      IDS_BRAVE_WALLET_CONFIRM_TRANSACTION_GAS_FEE},
-    {"braveWalletConfirmTransactionTransactionFee",
-     IDS_BRAVE_WALLET_CONFIRM_TRANSACTION_TRANSACTION_FEE},
```

Comment:
@onyb This shouldn't be removed by your PR.

Reply:
I don't recall why I removed it. Probably because it was not used at the time. Should've checked with a `git blame`. I see Douglas took care of it.